### PR TITLE
[ADD][12.0] google_maps

### PR DESCRIPTION
**Pull request by murtuzasaleh**
WIP

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 @wolfhall Can you please review this PR.

**Comment by max3903** [comment]
@murtuzasaleh The module must be at the root of the repo. Please remove WIP in the title and description if you are done.

File: google/google_maps/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        'views/res_partner.xml',
+        'views/res_config_settings.xml'
+    ],
+    'demo': [],
+    'images': ['static/description/thumbnails.png'],
+    'qweb': ['static/src/xml/widget_places.xml'],
+    'installable': True,
+    'uninstall_hook': 'uninstall_hook',
+    'maintainers': [
+        'gityopie',
```
Comment (murtuzasaleh) [inline google/google_maps/__manifest__.py]: @max3903 Is it fine or I have to remove it from here?

Comment (max3903) [reply google/google_maps/__manifest__.py]: @murtuzasaleh It depends on @gityopie.

@gityopie Can we count on you to help maintaining it?

Comment (gityopie) [reply google/google_maps/__manifest__.py]: @max3903  yes, you can. I'll try my best

Comment (murtuzasaleh) [reply google/google_maps/__manifest__.py]: > @murtuzasaleh The module must be at the root of the repo. Please remove WIP in the title and description if you are done.

@maxime As per the @wolfhall  comment I have added a module in OCA/geospatial/google/google_maps

Do you want to add directly to OCA/geospatial repo?

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh The module must be at the root of the repo. Please remove WIP in the title and description if you are done.

**Pull request closed without merging**